sklearn/linear_model/logistic.py

```diff
@@ -16,7 +16,8 @@
 import numpy as np
 from scipy import optimize, sparse
 
-from .base import LinearClassifierMixin, SparseCoefMixin, BaseEstimator
+from .base import (LinearClassifierMixin, SparseCoefMixin, BaseEstimator,
+                   LinearModel)
```

Comment:
Thanks. But with `LogisticRegression` extending `LinearModel`, I got into a MRO hell. I created a mixin now, but no need to review yet; it all still fails. :smile: